python/cuspatial/cuspatial/tests/conftest.py

```diff
@@ -92,56 +94,56 @@ def polys():
     return np.array(
         (
             (35, 36),
-            (37, 38),
-            (39, 40),
+            (38, 36),
+            (41, 39),
```

Comment:
So `polys` fixture is just listing all the coordinates from all polygons / multipolygons in `gs` fixture? I think the name of the fixture is quite confusing. At least a comment to help explaining it would be useful.

Reply:
Removed `polys`, it's redundant.